### ISSUE TITLE
lint-staged?

### ISSUE THREAD

**Issue by theKashey** [zzarcon/ts-react-toolbox#21]
`lint --fix` and `prettier` on commit? It would be just awesome :)

**Comment by zzarcon** [comment]
YES! I wanted to do that some time ago but never finished, we should add it 🗡
